scripts/region-checker/allowlist.js

```diff
@@ -4,9 +4,9 @@ var allowlist = {
         25,
         85,
         86,
-        201,
-        255,
-        256
+        202,
+        256,
+        257
```

Comment:
The build is failing because with error `Hard-coded regions detected. This should only be done if absolutely certain!`

<details>
<summary>Details</summary>

```console
Hard-coded regions detected. This should only be done if absolutely certain!
File: /config.js	Line 203:	*     accessKeyId: 'AKID', secretAccessKey: 'SECRET', region: 'us-west-2'
File: /config.js	Line 258:	*   'us-east-1' regional endpoints. This config is only applicable to S3 client.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! aws-sdk@2.813.0 region-check: `node ./scripts/region-checker/index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the aws-sdk@2.813.0 region-check script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-12T16_46_58_217Z-debug.log

> aws-sdk@2.813.0 region-check /codebuild/output/src826660540/src/github.com/aws/aws-sdk-js
> node ./scripts/region-checker/index.js

Hard-coded regions detected. This should only be done if absolutely certain!
File: /config.js	Line 203:	*     accessKeyId: 'AKID', secretAccessKey: 'SECRET', region: 'us-west-2'
File: /config.js	Line 258:	*   'us-east-1' regional endpoints. This config is only applicable to S3 client.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! aws-sdk@2.813.0 region-check: `node ./scripts/region-checker/index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the aws-sdk@2.813.0 region-check script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-12T16_46_58_431Z-debug.log
"npm run region-check" exited with code: 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! aws-sdk@2.813.0 test: `node ./scripts/composite-test.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the aws-sdk@2.813.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-12T16_46_58_442Z-debug.log
```

</details>

```suggestion
        203,
        257,
        258
```